styles/lazy-styles.css

```diff
@@ -1 +1,48 @@
-/* add global styles that can be loaded post LCP here */
\ No newline at end of file
+/* add global styles that can be loaded post LCP here */
+
+/* Social Dialog Share */
+dialog.social-share {
+    border: none;
+    padding: 0;
+    border-radius: 10px;
+}
+
+dialog.social-share > div {
+    padding: 2rem;
+    border: 2px solid var(--link-color-orange);
+    border-radius: 10px;
+    background-color: var(--color-white);
+    min-width: 500px;
```

Comment:
Don't need this as it is creating H-Scroll bar in mob view, without it view are good on every device

Uploading Screen Recording 2024-04-01 at 3.40.01 PM.mov…